lib/pages/game.dart

```diff
@@ -57,26 +57,40 @@ class _GameState extends State<Game> {
       backgroundColor: const Color.fromARGB(255, 31, 16, 42),
       body: Column(
         children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Padding(
+                padding: const EdgeInsets.all(20.0),
+                child: Text("Punkte: $points",
+                    style: const TextStyle(
+                      color: Colors.white,
+                      fontWeight: FontWeight.bold,
+                      fontSize: 30,
+                    )),
+              ),
+              Padding(
+                  padding: const EdgeInsets.all(20.0),
+                  child: IconButton(
+                      onPressed: () => endGame(),
+                      icon: const Icon(Icons.highlight_remove_sharp,
+                          size: 50, color: Colors.white)))
+            ],
+          ),
           Row(
```

Comment:
this could be moved into the appBar to resolve the problem that the game moves when no powerups are available